trigger_aws.yml

```diff
@@ -63,19 +64,16 @@ phases:
       - echo "Navigate to test package directory"
       - cd $DEVICEFARM_TEST_PACKAGE_PATH
       - echo "Start Appium Python test"
-      - bin/py.test --collect-only tests/android/tests/test_android_new_landing.py tests/android/tests/test_android_login.py tests/android/tests/test_android_main_dashboard.py tests/android/tests/test_android_my_courses_list.py tests/android/tests/test_android_course_discovery.py tests/android/tests/test_android_register.py 
-
       # By default, the following command is used by Device Farm to run your Appium Python test.
       # The goal is to run all your tests files in the test package.
       # Alternatively, You may specify your customized command.
       # Note: For most use cases, the default command works fine.
       # Please refer "https://docs.pytest.org/en/latest/usage.html" for more options on running pytests from command line.
-      - bin/py.test -v tests/android/tests/test_android_new_landing.py tests/android/tests/test_android_login.py tests/android/tests/test_android_main_dashboard.py tests/android/tests/test_android_my_courses_list.py tests/android/tests/test_android_course_discovery.py tests/android/tests/test_android_register.py --html=$DEVICEFARM_LOG_DIR/report.html      
-
-  # The post test phase includes are commands that are run after your tests are executed.
+      - bin/py.test --collect-only tests/android/tests/test_android_new_landing.py tests/android/tests/test_android_login.py tests/android/tests/test_android_main_dashboard.py tests/android/tests/test_android_my_courses_list.py tests/android/tests/test_android_course_discovery.py tests/android/tests/test_android_register.py
+      - bin/py.test -v tests/android/tests/test_android_new_landing.py tests/android/tests/test_android_login.py tests/android/tests/test_android_main_dashboard.py tests/android/tests/test_android_my_courses_list.py tests/android/tests/test_android_course_discovery.py tests/android/tests/test_android_register.py --html=$DEVICEFARM_LOG_DIR/report.html
+      # The post test phase includes are commands that are run after your tests are executed.
```

Comment:
need to update the doc-string, as not supported by current changes.

Reply:
fixed